tests/modules/worksheets/test_worksheet.py

```diff
@@ -145,3 +181,36 @@ def test_update_worksheet(client, worksheet, group):
     )
 
     assert response.data.updateWorksheet.label == 'change label'
+
+
+def test_create_query_download_url(client, worksheet, env_fixture):
+    response = client.query(
+        """
+        mutation CreateWorksheetQueryResultDownloadUrl($input:WorksheetQueryResultDownloadUrlInput){
+            createWorksheetQueryResultDownloadUrl(input:$input){
+                queryType
+                sqlBody
+                AthenaQueryId
+                region
+                AwsAccountId
+                ElapsedTimeInMs
+                created
+                downloadLink
+                OutputLocation
+                expiresIn
+                fileFormat
+            }
+        }
+        """,
+        input={
+            'worksheetUri': worksheet.worksheetUri,
+            'athenaQueryId': '123',
+            'fileFormat': 'csv',
+            'environmentUri': env_fixture.environmentUri,
+        },
+    )
+
+    expires_in = datetime.strptime(response.data.createWorksheetQueryResultDownloadUrl.created, '%Y-%m-%d %H:%M:%S.%f')
+    assert response.data.createWorksheetQueryResultDownloadUrl.downloadLink is not None
+    assert response.data.createWorksheetQueryResultDownloadUrl.fileFormat == 'csv'
+    assert expires_in > datetime.utcnow()
```

Comment:
After implementing the permission check, see comment in the service file, we need to implement a test scenario for the unauthorized not-happy-path